hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Server.java

```diff
@@ -1351,6 +1357,16 @@ private class ResponseParams {
 
     @Override
     public String toString() {
+      boolean isCallerContextEnabled = conf.getBoolean(
```

Comment:
I don't think this is a good idea to get the value from conf every time. And `HADOOP_CALLER_CONTEXT_ENABLED_KEY` is used to control whether output callerContext into audit log. We plan use this key here?

Reply:
Here, the essence is to obtain the real client ip, and it is not a problem to use HADOOP_CALLER_CONTEXT_ENABLED_KEY.